portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,9 +19,11 @@
 
   private final String name;
   private final String comment;
+  private final int sentiment;
 
-  public Comment(String name, String comment) {
+  public Comment(String name, String comment, int sentiment) {
     this.name = name;
     this.comment = comment;
+    this.sentiment = sentiment;
```

Comment:
is this change meant for this PR? Additions for sentiment analysis might make more sense in a subsequent PR focused on that feature.

Reply:
Ack